intl/msg_hash_us.h

```diff
@@ -1597,13 +1597,13 @@ MSG_HASH(MENU_ENUM_SUBLABEL_CPU_CORES,
 MSG_HASH(MENU_ENUM_SUBLABEL_FPS_SHOW,
       "Displays the current framerate per second onscreen.")
 MSG_HASH(MENU_ENUM_SUBLABEL_INPUT_HOTKEY_BINDS,
-      "Configure hotkey settings.")
+      "Change settings for hotkey.")
 MSG_HASH(MENU_ENUM_SUBLABEL_INPUT_MENU_ENUM_TOGGLE_GAMEPAD_COMBO,
       "Gamepad button combination to toggle menu.")
 MSG_HASH(MENU_ENUM_SUBLABEL_INPUT_SETTINGS,
-      "Adjusts settings for joypads, keyboard and mouse.")
+      "Change settings for joypads, keyboard and mouse.")
 MSG_HASH(MENU_ENUM_SUBLABEL_INPUT_USER_BINDS,
-      "Configure controls for this user.")
+      "Change controls for this user.")
 MSG_HASH(MENU_ENUM_SUBLABEL_LOG_VERBOSITY,
```

Comment:
This is to create consistent with other options. Also, I notice that some settings could do without description (like Language for one).